controllers/provisioners/eks/eks.go

```diff
@@ -108,6 +110,19 @@ func (ctx *EksInstanceGroupContext) GetInstanceGroup() *v1alpha1.InstanceGroup {
 	return &v1alpha1.InstanceGroup{}
 }
 
+func (ctx *EksInstanceGroupContext) GetOsFamily() string {
+	if ctx != nil && ctx.InstanceGroup != nil {
```

Comment:
We probably don't need a nil check here because you cannot get to this code path without an InstanceGroup or context object being populated.

Very minor, but might be a bit nicer to use 
```golang
var (
  instanceGroup  = ctx.GetInstanceGroup()
  labels = instanceGroup.GetLabels()
)
```
And also have more safety around getting the label:

```golang
if ok, _ := labels[OsFamilyLabel]; ok {
  return labels[OsFamilyLabel]
} 
return "default"  // return "" is will also be fine here
```